src/components/SRNEntry.vue

```diff
@@ -179,50 +180,51 @@ export default {
     },
-    /* removes an element in the array at a given index. 
-    @param {Number} - index - the index of the input box where - button is clicked. */
+    /** Removes an element in the array at a given index.
+     * @param {Number} index - the index of the input box where - button is clicked
+     */
     removeInputField(index) {
       this.userIDList.splice(index, 1);
     },
-    // sets the invalid input message to null (default)
+    /** Sets the invalid input message to null (default) */
     resetInvalidInputMessage() {
       this.invalidInputMessage = "";
     },
-    // sets the invalid login message to null (default)
+    /** Sets the invalid login message to null (default) */
     resetInvalidLoginMessage() {
       this.invalidLoginMessage = "";
     },
-    // sets the temp valid flag to false (default) for authentication of a possible next user
+    /** Sets the temp valid flag to false (default) for authentication of a possible next user */
     resetValidFlag() {
       this.isCurrentUserValid = false;
     },
-    // sets the valid key of the latest user to the value of the temp flag. The temp flag contains the value returned by the backend
+    /** Sets the valid key of the latest user to the value of the temp flag. The temp flag contains the value returned by the backend */
     setValidFlag() {
-      this.getLatestEntry["valid"] = this.isCurrentUserValid;
+      this.latestEntry["valid"] = this.isCurrentUserValid;
     },
-    //resets the userID key of an element in the array at a particular index.
-    resetEntry(index){
-        this.userIDList[index]["userID"] = "";
+    /** Resets the userID key of an element in the array at a particular index */
+    resetEntry(index) {
+      this.userIDList[index]["userID"] = "";
     },
-    /* this function is called whenever the + button is clicked. The most recent typed entry is authenticated against the database. 
-    - if it does not exist and it is the first time being authenticated, handleIncorrectEntry() is called. 
-    - if the entry exists or if it is being authenticated the second time, the 'valid' flag of the entry is set to whatever the value is returned by the backend. 
-      Also, isCurrentUserValid and validateCount are reset to their default values, so that a possible next entry can be processed. */
+    /** This function is called whenever the + button is clicked.
+     * The most recent typed entry is authenticated against the database.
+     */
     async addField() {
-      const latestUserID = parseInt(this.getLatestEntry["userID"]);
+      const latestUserID = parseInt(this.latestEntry["userID"]);
       if (!isNaN(latestUserID)) {
         await this.authenticateSRN(latestUserID);
-        if(!this.isCurrentUserValid && this.validateCount == 1){
+        if (!this.isCurrentUserValid && this.validateCount == 1) {
           this.handleIncorrectEntry(latestUserID);
         }
         if (this.isCurrentUserValid || this.validateCount > 1) {
```

Comment:
Not able to comment on the actual code so commenting here

Taking a look at this if block, notice the third line
`this.resetValidFlag`
Is it a method? If it's a method, there's no parenthese to invoke it?